core/modules/czar/CzarConfig.cc

```diff
@@ -38,6 +41,22 @@ namespace {
 
 LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.CzarConfig");
 
+void QservLogger(struct timeval const& mtime,
+                 unsigned long         tID,
+                 const char*           msg,
+                 int                   mlen) {
+
+    static log4cxx::spi::LocationInfo xrdLoc("client", "<xrdssi>", 0);
+    static LOG_LOGGER myLog = LOG_GET("lsst.qserv.xrdssi.msgs");
+
+    if (myLog.isInfoEnabled()) {
+        std::string theMsg(msg, mlen);
```

Comment:
Looking at the log it seems that message comes from xrootd with a trailing newline character. To avoid empty lines in the log can you strip that newline character here?